### DISCUSSION TITLE
Unable to resolve the error AutoConfiguredOpenTelemetrySdk found on classpath but automatic configuration is disabled

### DISCUSSION THREAD

**Discussion by yatamraghu7** [open-telemetry/opentelemetry-java-instrumentation#12429] [Q&A]
Hi @zeitlinger

I am creating this ticket as reference for the discussion that we did on https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/11765

I tried using 2.8.0 version of BOM but still I can see that error statement 

AutoConfiguredOpenTelemetrySdk found on classpath but automatic configuration is disabled. To enable, run your JVM with -Dotel.java.global-autoconfigure.enabled=true

Even I have removed the b3 trace propagators in my applicaiton properties and getting the error. For now please ignore 
"opentelemetry-spring-webflux-5.3 is included in the stater already "

Can you tell us why in the first place the autoconfiguration provided by opentelemetry SDK is disabled without doing anything from our end ?

I wanted to get a tracer bean but as I am using spring boot starter opentelemetry I understand that it wont provide the bean out of the box I need to get it from GlobalOpenTelemetry but when I try to do that I am getting "000000000000000000000000" as my traceId.

Thanks

Thanks

**Comment by zeitlinger** [comment]
please provide an updated repo and the steps I need to do to see 000000000000000000000000
